api_docs/construct-narrow.md

```diff
@@ -45,6 +45,13 @@ operator, search the current user's personal message history. See
 [searching shared history](/help/search-for-messages#searching-shared-history)
 for details.
 
+**Changes**: In Zulip 7.0 (feature level 177), support was added for
+  three narrows related to direct messages: `is:dm`, `dm` and
+  `dm-including`. The `dm` operator replaced and deprecated the
```

Comment:
nit: I think of a "narrow" as meaning specifically a complete set of filters one might send to /api/get-messages. It looks like this page currently consistently uses "narrow" with that meaning too.

So `is:dm` is a narrow, but more generally it's a filter (an operator-operand pair) one can include in a narrow. And `dm` and `dm-including` are operators, which are building blocks of narrows but not narrows on their own.

If it were just `dm` and `dm-including`, I'd say "two operators". For all three of these, I think "three filters" would be fine. (Strictly it's three *kinds* of filters: `is:dm` plus `-is:dm`, and filters with the operators `dm` and `dm-including`. But that's more verbose and probably not helpful.)

Reply:
This makes sense. Updated version uses "filter" in general and "operator" for sentences that are about `dm` or `dm-including` specifically.